### PR TITLE
raghunadhpokkalath/c02-network02

### PR DESCRIPTION
**Pull request by raghunadhpokkalath**
# Description

Exercise (e.g. C01-E01): c02-network02

Student Name:Raghunadh Pokkalath

> A brief explanation of what has been done.

## Type of change

- [x] Exercise Submission (one exercise per PR)
  - [x] This PR is just for **one** exercise of a class
  - [x] PR name follows the pattern `<github-username>/<exercise-number>`
  - [x] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [x] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/02class/exercises/c02-network02/raghunadhpokkalath/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# c02-network02
+
+## Commands Execution Output
+
+- Answers for `10.0.0.0/16`:
+
+|Subnet|Network|Host Range|Usable Hosts|Broadcast|AWS Reserved|
```
Comment (christina-roperto) [inline classes/02class/exercises/c02-network02/raghunadhpokkalath/README.md:7]: I think you got the host range and usable hosts header inverted :)

File: classes/02class/exercises/c02-network02/raghunadhpokkalath/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+|-|-|-|-|-|-|
+|subnet-a|10.0.0.0/18 | 16379 |10.0.0.4 - 10.0.63.254 | 10.0.63.255|10.0.0.1,10.0.0.2,10.0.0.3 
+|subnet-b|10.0.64.0/18|16379|10.0.64.4 - 10.0.127.254 |10.0.127.255 |10.0.64.1,10.0.64.2,10.0.64.3 
+|subnet-c|10.0.128.0/18|16379 |10.0.128.4 - 10.0.191.254 | 10.0.191.255 |10.0.64.1,10.0.64.2,10.0.64.3
```
Comment (christina-roperto) [inline classes/02class/exercises/c02-network02/raghunadhpokkalath/README.md:11]: the aws reserved IPs for subnet b and c are the same. Could you fix the subnet c ones?

File: classes/02class/exercises/c02-network02/raghunadhpokkalath/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+|subnet-d|10.0.192.0/18 | 16379|10.0.192.4 - 10.0.255.254  |10.0.255.255 |10.0.192.1,10.0.192.2,10.0.192.3
+
+- Answers for `192.168.0.0/24`:
+
+|Subnet|Network|Host Range|Usable Hosts|Broadcast|AWS Reserved|
+|-|-|-|-|-|-|
+|subnet-a|192.168.0.0/26 |192.168.0.4 - 192.168.0.62 |59 |192.168.0.63 |192.168.0.1,192.168.0.2,192.168.0.3
+|subnet-b|192.168.0.64/26 |192.168.0.68 - 192.168.0.126 | 59 | 192.168.0.127|192.168.0.65,192.168.0.66 ,192.168.0.67
+|subnet-c|192.168.0.127/26 |192.168.0.131 - 192.168.0.190 | 59 | 192.168.0.191|192.168.0.128,192.168.0.129,192.168.0.130
```
Comment (christina-roperto) [inline classes/02class/exercises/c02-network02/raghunadhpokkalath/README.md:20]: hi raghu, you might want to check the network IP for subnet c. 192.168.0.127 is already used as broadcast for subnet b.

File: classes/02class/exercises/c02-network02/raghunadhpokkalath/README.md (hunk 1 above)
Comment (raghunadhpokkalath) [reply classes/02class/exercises/c02-network02/raghunadhpokkalath/README.md:7]: Hi @christina-roperto thanks i have corrected it

File: classes/02class/exercises/c02-network02/raghunadhpokkalath/README.md (hunk 2 above)
Comment (raghunadhpokkalath) [reply classes/02class/exercises/c02-network02/raghunadhpokkalath/README.md]: sorry typo i have fixed it

File: classes/02class/exercises/c02-network02/raghunadhpokkalath/README.md (hunk 3 above)
Comment (raghunadhpokkalath) [reply classes/02class/exercises/c02-network02/raghunadhpokkalath/README.md]: yes @christina-roperto i missed it corrected now

**Pull request merged**